gobblin-hive-registration/src/main/java/gobblin/hive/metastore/HiveMetaStoreBasedRegister.java

```diff
@@ -267,6 +271,14 @@ private void addOrAlterPartition(IMetaStoreClient client, Table table, Partition
     }
   }
 
+  private String stringifyPartition(Partition partition) {
+    if (log.isDebugEnabled()) {
```

Comment:
It is just to avoid repeating the log line. If the logger is in debug mode, will print a lot more information about the table.